build.sbt

```diff
@@ -62,11 +62,12 @@ lazy val aggregatedProjects: Seq[ProjectReference] = userProjects ++ List[Projec
   httpJmhBench,
   billOfMaterials)
 lazy val root = Project(
-  id = "pekko-http-root",
+  id = "pekko-http",
```

Comment:
This is to be consistent with pekko-core, see https://github.com/apache/incubator-pekko/blob/main/build.sbt#L88-L92